### ISSUE TITLE
Styles for Current month

### ISSUE THREAD

**Issue by n1kko777** [nickeljew/react-month-picker#62]
Please, make ability to style current month.

**Comment by nickeljew** [comment]
you may override the style in your css/scss

**Comment by n1kko777** [comment]
How I can access/select current month ?

**Comment by nickeljew** [comment]
BTW, what you meant about the current month? the active one?
Why don't you try to use chrome's inspect page to locate the element to find the CSS class path??

.month-picker > .rmp-container .rmp-popup.light .rmp-pad li.active
Is that one what you looking for?

**Comment by n1kko777** [comment]
No, no, no! I mean the month which is going right now (today’s month) 😊

**Comment by nickeljew** [comment]
Oh, I see, but it's not supported for now. you may rewrite the component for your purpose, for I don't think it's a common requirement.

**Comment by nickeljew** [comment]
but I think there is a way you can try.
check the "current" year value of the panel, if it's this year, then you may locate the LI tag at the index to match the "current" month.

**Comment by n1kko777** [comment]
Yeah, it’s my way
